config: return a noop SDK if disabled is set to true (#6185)
open-telemetry/opentelemetry-go-contrib: CHANGELOG.md

### Changed

- The function signature of `NewLogProcessor` in `go.opentelemetry.io/contrib/processors/minsev` has changed to accept the added `Severitier` interface instead of a `log.Severity`. (#6116)

> - `NewSDK` now returns a no-op SDK if `disabled` is set to `true` via configuration. (#6185)

Reviewer: We usually specify the exact package the change it made in, as there are so many in this repository.

```suggestion
- `NewSDK` in `go.opentelemetry.io/contrib/config` now returns a no-op SDK if `disabled` is set to `true`. (#6185)
```
